cmake/cyclonedds/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 # SPDX-License-Identifier: Apache-2.0
-cmake_minimum_required(VERSION 3.5)
```

Comment:
Instead of setting in every file the required cmake version, couldn't we add a generic cmake options file which we include where this is set.
I would bet that the reviewers here check only the files which you touched and not mention the files you may have forgotten to adjust.

Reply:
According to this little post this is not possible: https://stackoverflow.com/questions/57638044/can-you-have-a-common-cmake-minimum-included-from-another-file

But maybe some other wise developer knows more stuff?! Or tricks?

Reply:
For now it is fine to set them in each file, the effort is not that much when doing a search & replace over the project.
I don't see any special trick for circumventing this, even wrapping this into a custom function does not work since the version is not set in the calling scope.